assets/js/autocomplete/suggestions.js

```diff
@@ -151,7 +151,13 @@ function findIn (elements, term, categoryName) {
  * @returns {string} Text with matching part highlighted with html <em> tag.
  */
 function highlight (match) {
-  return match.input.replace(match, `<em>${match[0]}</em>`)
+  const EmOpen = '###EM_OPEN###'
+  const EmClose = '###EM_CLOSE###'
+
+  let result = match.input
+  result = result.replace(match, EmOpen + match[0] + EmClose)
+  result = escapeHtmlEntities(result)
+  return result.replace(EmOpen, '<em>').replace(EmClose, '<\/em>')
```

Comment:
I think this is not needed - I'm not getting any escpaing issues with the default implementation.

Reply:
The tricky part is when the search term contains a special html entity, such as `<` or `>`. Could you please try them with that. The changes in this functions is to deal with that edge case.

Reply:
I see - I can break it with `">` thought we already account for that, but it's not the case.

![image](https://user-images.githubusercontent.com/1810732/90411990-d6258000-e0ac-11ea-88a6-203b7ae24564.png)

thanks!